docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -483,26 +484,49 @@ def source(self, follow_additional_fields=True, document_type="pdf"):
                 content += ")"
               content += "\n" 
             content += "\n"
+            
+        elif self.type == 'main order':
+          lines = [
+            "mandatory: True",
+            "id: main_order_" + self.interview_label,
+            "code: |",
+            "  " + "# Controls the flow of the basic building blocks of the",
+            "  " + "# interview. To use this interview in another interview",
+            "  " + "# delete the `mandatory: True` specifier or this whole block.",
+            "  " + self.intro + "  # Organization intro screen/splash screen",
+            "  " + "# Introduction to this specific interview",
+            "  " + self.interview_label + "_intro",
+            "  " + "# Trigger the whole interview order block to control question order",
+            "  " + self.interview_label,
+            "  " + "signature_date",
+            # Save a snapshot of interview answers. 
+            # We only want a few anonymous variables
+            "  " + "# Save (anonymized) interview statistics.",
+            "  " + "store_variables_snapshot(data={'zip': users[0].address.zip})",
+            "  " + self.interview_label + "_preview_question  # Pre-canned preview screen",
+            "  " + "basic_questions_signature_flow",
```

Comment:
Was easier to give them all the indent separately so it was easy to see when things were indented.

Reply:
Why does this need its own block type? Isn't it just a code block with a custom ID?

Reply:
Does this have to do with the formatting I was commenting on?

Reply:
No. It looks like Github put this in its own thread but I didn't intend to reply to your comment. My comment is the whole block being a new question type vs using existing code block question type.